dotnet-grad/Controllers/UserController.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using dotnet_grad.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace dotnet_grad.Controllers
+{
+  [ApiController]
+  [Route("api/[controller]")]
+  public class UserController : ControllerBase
+  {
+    TestContext testContext;
+    public UserController(TestContext _testContext)
+    {
+      testContext = _testContext;
+    }
+
+    [HttpGet("GetAllUsers")]
+    public List<UserModel> getUsers()
```

Comment:
Naming of the method is not according to standard. It should start with an uppercase